Gemfile.lock

```diff
@@ -8,33 +8,180 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    activemodel (6.1.5.1)
-      activesupport (= 6.1.5.1)
-    activerecord (6.1.5.1)
-      activemodel (= 6.1.5.1)
-      activesupport (= 6.1.5.1)
-    activesupport (6.1.5.1)
+    actioncable (7.0.2.4)
```

Comment:
I'd vote for removing the `Gemfile.lock` from the repo, as this is a gem, so we shouldn't lock gems to any specific version.